experiments/scripts/run_tune.py

```diff
@@ -102,5 +107,7 @@ def run(config_path: str) -> None:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("config_path", help="The full path to the YAML config file.")
+    parser.add_argument("config_overrides", help="Any overriding parameters as a JSON.")
     args = parser.parse_args()
-    run(args.config_path)
+    parsed_json_overrides = json.loads(args.config_overrides)
+    run(args.config_path, parsed_json_overrides)
```

Comment:
is "config_overrides" optional here like it says it should be in `run`? It looks like it would have to be passed in here or `json.loads()` would not work?